apps/kol-amelamdim/components/navbar/Navbar.tsx

```diff
@@ -123,6 +116,35 @@ export const Navbar = () => {
               </Button>
             </div>
           )}
+          <Select
+            labelId="demo-simple-select-label"
+            id="demo-simple-select"
+            value={i18n.language || 'he'}
+            onChange={handleLanguageChange}
+            sx={{
+              height: '30px',
+              width: '40px',
+              '.MuiOutlinedInput-notchedOutline': { border: 'none' },
+            }}
```

Comment:
The icon needs to be aligned with the logout/sign in button (need to decrease it a little)

<img width="194" alt="image" src="https://user-images.githubusercontent.com/110565257/194622280-6cf50b93-2c2d-4e06-a891-f1a7ff140d97.png">